keylime/cloud_verifier_common.py

```diff
@@ -209,7 +212,9 @@ def process_quote_response(agent, json_response):
                                      ima_measurement_list,
                                      agent['allowlist'],
                                      hash_alg,
-                                     ima_keyring)
+                                     ima_keyring,
+                                     mb_measurement_list,
+                                     {})
```

Comment:
Just curious what the empty dict is for?

Reply:
in the subsequent PRs, it will be changed to the "measured boot reference state" supplied by the `tenant` CLI